interface/usergroup/docs_validation.php

```diff
@@ -0,0 +1,87 @@
+<?php
```

Comment:
This needs a documentation header then I think it can be merged. @apooravc look at a file like the style.php the naveen did in the color picker pr